### PR TITLE
Transform `getAboutUsData` function into `aboutUsData` object

### PR DESCRIPTION
**Pull request by mdubus**
# Pull Request

## What does this PR do?
Since the `getXXX` function is legacy, we do not need to keep this format anymore.
The `getAboutUsData` function as been transformed into a `aboutUsData` object.

## PR checklist
Please check if your PR fulfills the following requirements:
- [x] Does this PR fix an existing issue, or have you listed the changes applied in the PR description (and why they are needed)?
- [x] Have you read the contributing guidelines?
- [x] Have you made sure that the title is accurate and descriptive of the changes?

Thank you so much for contributing to Meilisearch!

### CODE REVIEW

File: src/pages/about-us.js [hunk 1]
```diff
@@ -1,10 +1,10 @@
 import Head from 'components/Head'
-import getAboutUsData from '../../data/about-us'
+import aboutUsData from '../../data/about-us'
```
Comment (bidoubiwa) [inline src/pages/about-us.js:2]: another way is like this:

```suggestion
import { meta, hero } from '../../data/about-us'
```

But maybe you prefer to know where the variables are coming from lower in the code

**Review by bidoubiwa — COMMENTED** [review]
LGTM

File: src/pages/about-us.js (hunk 1 above)
Comment (mdubus) [reply src/pages/about-us.js:2]: Yes absolutely, I'll keep it this way but thanks anyway for the suggestion ! 😊

**Comment by mdubus** [comment]
bors merge

**Pull request merged**